manim/mobject/coordinate_systems.py

```diff
@@ -665,16 +746,32 @@ def get_lines_parallel_to_axis(
                     lines2.add(new_line)
         return lines1, lines2
 
-    def get_center_point(self):
+    def get_center_point(self) -> np.ndarray:
+        """Gets the center point from coordinates.
+
+        Returns
+        -------
+        np.ndarray
+            Returns the center point.
+        """
         return self.coords_to_point(0, 0)
 
     def get_x_unit_size(self):
+        # not yet implemented ?
         return self.get_x_axis().get_unit_size()
 
     def get_y_unit_size(self):
+        # not yet implemented ?
         return self.get_x_axis().get_unit_size()
 
-    def get_axes(self):
+    def get_axes(self) -> VGroup:
+        """Gets the pair of axes.
+
+        Returns
+        -------
+        :class:`~.VGroup`
```

Comment:
same

Reply:
I am not sure I understand what you mean here.

Reply:
I think eulertour was talking about omitting the word `Returns` in the [next line](https://github.com/ManimCommunity/manim/pull/1465/files#diff-b6144e02b604d43f8149021177941b176f959cb360e061aace916f5ddcc4742aR775).